mathquill4quill.js

```diff
@@ -277,8 +286,22 @@ window.mathquill4quill = function(dependencies) {
         button.setAttribute("class", "mathquill4quill-history-button");
       }
 
-      function applyHistoryContainerStyles(container) {
-        container.setAttribute("class", "mathquill4quill-history-container");
+      function applyHistoryInnerContainerStyles(container) {
+        container.setAttribute("class", "mathquill4quill-history-inner-container");
```

Comment:
Nit: apply `./node_modules/.bin/prettier --write "**/*.js"` so that CI passes.
```suggestion
        container.setAttribute(
          "class",
          "mathquill4quill-history-inner-container"
        );
```